Silk.Core/Commands/General/AvatarCommand.cs

```diff
@@ -62,8 +62,7 @@ private static DiscordEmbedBuilder DefaultAvatarEmbed(CommandContext ctx)
         {
             return new DiscordEmbedBuilder()
                 .WithColor(DiscordColor.CornflowerBlue)
-                .WithFooter("Silk", ctx.Client.CurrentUser.AvatarUrl)
-                .WithTimestamp(DateTime.Now);
+                .WithFooter("Silk", ctx.Client.CurrentUser.AvatarUrl);
```

Comment:
Feel free to remove this footer entirely, or change it to `$"Requested by {ctx.User.Id}"`